themes/startup-theme-components/templates/SilverStripe/StartupThemeComponents/Elemental/Block/ImageTextBlock.ss

```diff
@@ -0,0 +1,27 @@
+<div class="image-text-block">
+    <div class="container image-text-block__container $VariantClass">
+        <div class="image-text-block__left-column">
+            <div class="image-text-block__image-wrap">
+                <img src="$ImageTextBlockImage.URL" class="image-text-block__image" alt="$ImageTextBlockImage.AltText"
+                     loading="lazy"/>
+            </div>
+        </div>
+        <div class="image-text-block__right-column">
+            <% if $Title && $ShowTitle %>
+                <% if $Content %>
+                    <h2>$Title</h2>
+                <% else %>
+                    <h1>$Title</h1>
+                <% end_if %>
```

Comment:
This logic doesn't match up with the AC:

> Title (required - NB if first block on blocks page without hero area this will be a H1)

The h1 should appear if it's the first block on the page and `ShowHero` is false.

Reply:
Thank you @BrookeNZ , initially I did not get this Ac criteria